src/digital-asset/dto/digital-asset.dto.ts

```diff
@@ -96,18 +96,22 @@ export async function toNftDto(nft: NftInput) {
     rarity: findRarityTrait(offChainMetadata),
     comicName: offChainMetadata.collection.family,
     comicIssueName: offChainMetadata.collection.name,
-    comicIssueId: nft.collectionNft?.comicIssueId,
+    comicIssueId: asset.collectionNft?.comicIssueId,
```

Comment:
We are having some problems with types here. This will always be `undefined`. We should take this from asset.metadata.collection.comicIssueId